extensions/pkg/webhook/registration.go

```diff
@@ -66,73 +84,70 @@ func BuildWebhookConfigs(
 	mode, url string,
 	caBundle []byte,
 ) (
-	seedWebhookConfig client.Object,
+	seedWebhookConfig Configs,
 	shootWebhookConfig *admissionregistrationv1.MutatingWebhookConfiguration,
 	err error,
 ) {
 	var (
 		exact       = admissionregistrationv1.Exact
 		sideEffects = admissionregistrationv1.SideEffectClassNone
+		shootMode   = ModeURLWithServiceName
 
-		seedWebhooks  []admissionregistrationv1.MutatingWebhook
 		shootWebhooks []admissionregistrationv1.MutatingWebhook
 	)
 
+	if mode == ModeURL {
+		shootMode = ModeURL
+	}
+
 	for _, webhook := range webhooks {
 		var rules []admissionregistrationv1.RuleWithOperations
 
 		for _, t := range webhook.Types {
 			rule, err := buildRule(c, t)
 			if err != nil {
-				return nil, nil, err
+				return seedWebhookConfig, nil, err
```

Comment:
Why is this needed?

Reply:
Hm, it has to return something?! Earlier it was a pointer, hence return `nil`, but this has been changed because it simplifies the overall handling a bit.